lib/tapioca/sorbet_ext/name_patch.rb

```diff
@@ -1,12 +1,18 @@
 # typed: true
 # frozen_string_literal: true
 
+# We need sorbet to compile this signature before applying the patch
+# to avoid an infinite loop.
+::Tapioca::Reflection.qualified_name_of(String)
```

Comment:
Open to suggestions of a less bad way of doing this.

Reply:
Does running `T::Utils.run_all_sig_blocks` work instead? Alternatively, we could mark the sig of `qualified_name_of` as `checked(:never)` as well, if that helps.